### ISSUE TITLE
Terralith creates bogus structures that pollute the menu

### ISSUE THREAD

**Issue by rjp** [mrkite/minutor#359]
<img width="476" alt="Screenshot 2023-06-09 at 11 45 40" src="https://github.com/mrkite/minutor/assets/10793/6f4f34e2-e6a0-42ee-b51f-d4f4bd0a3034">

They are weird structures since they have a zero-sized bounding box.  e.g. `terralith:zpointer/minecraft_lush_caves -> [2256, 32, 2688, 2256, 32, 2688]`

Had a go at fixing this in two ways - first was adding a JSON file containing structure prefixes to ignore (`Structure.terralith:zpointer`).  Second was ignoring structures with a zero-sized bounding box.

Don't know whether one or the other covers more mods or whether both are needed.  Happy to create a PR with either / both.

**Comment by mrkite** [comment]
Ignoring zero-sized bounding boxes would be better than to single out a
specific mod.

**Comment by EtlamGit** [comment]
And ignoring zero sized Structures should be a one liner in `generatedstructure.cpp` (if at line 234)

By the way, you should update to latest Minutor. Structure menu is nested now.

**Comment by rjp** [comment]
> By the way, you should update to latest Minutor. Structure menu is nested now.

Yeah, I noticed that whilst building the app for testing my changes.  Much appreciated change!
